### PR TITLE
Update hacking.md

### PR DESCRIPTION
**Pull request by miccal**

### CODE REVIEW

File: doc/hacking.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ Since upgrading is a core feature of every package manager, the implementation o
...
+## Homebrew and Homebrew-Cask
 
-Homebrew-cask is independent of Homebrew as a project.
+Homebrew-Cask is independent of Homebrew as a project.
 
-The Homebrew-cask CLI is implemented as a Homebrew subcommand, so we try to match semantics wherever possible. That means that similar functionality should have similar flags and parameters.
+The Homebrew-Cask CLI is implemented as a Homebrew subcommand, so we try to match semantics wherever possible. That means that similar functionality should have similar flags and parameters.
 
-However, very little backend code is shared between the two projects. The Homebrew codebase is based on how Homebrew Formulae work, and our Casks are very much unlike Formulae.
+However, very little backend code is shared between the two projects. The Homebrew codebase is based on how Homebrew Formulae work, and our Casks are very different to Formulae.
```
Comment (vitorgalvao) [inline doc/hacking.md]: “different *from*”?

File: doc/hacking.md [hunk 2]
```diff
@@ -27,13 +27,13 @@ Since upgrading is a core feature of every package manager, the implementation o
...
 
 Because our backend needs are so different, we are discussing whether we should completely separate our codebase from Homebrew’s (see [#5080](https://github.com/caskroom/homebrew-cask/issues/5080)).
```
Comment (tsparber) [inline doc/hacking.md]: Current discussion tries to merge https://github.com/caskroom/homebrew-cask/issues/14384

Comment (vitorgalvao) [inline doc/hacking.md]: Nice point. This should also be updated to address that.

File: doc/hacking.md (hunk 1 above)
Comment (miccal) [inline doc/hacking.md]: Better.

File: doc/hacking.md (hunk 2 above)
Comment (miccal) [inline doc/hacking.md]: I will leave that to someone else more experienced.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 0adf277fded03f0b550bf889a1690887e1e2f1cb. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**